src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/coding/ConstructorsDeclarationGroupingCheck.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<checkstyle-metadata>
+   <module>
+      <check fully-qualified-name="com.puppycrawl.tools.checkstyle.checks.coding.ConstructorsDeclarationGroupingCheck"
+             name="ConstructorsDeclarationGrouping"
+             parent="com.puppycrawl.tools.checkstyle.TreeWalker">
+         <description>&lt;p&gt;
+ Checks that all constructors are grouped together.
+ If there is any code between the constructors
+ then this check will give an error.
+ The check identifies and flags any constructors that are not grouped together.
+ The error message will specify the line number from where the constructors are separated.
+ Comments between constructors are allowed.
```

Comment:
Please use language like `log` and `violation`, not `flag` and `error`. Run this through ChatGPT and eliminate redundant language here.

Reply:
done, here is the new description:

https://github.com/checkstyle/checkstyle/blob/580590b24455d2983729cc3223ab87876de22d1e/src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ConstructorsDeclarationGroupingCheck.java#L34-L39